src/components/TheNav.vue

```diff
@@ -85,47 +86,47 @@ export default {
           content: 'Home',
           conditionOne: true,
           conditionTwo: false,
-          function: ''
+          callback: ''
```

Comment:
it has to be a function

Reply:
or it can be undefined

Reply:
(not defined)

Reply:
So, you're saying I should omit this property entirely when I don't need any function? Won't it cause an error?

Reply:
It already causes an error as string is not a function

Reply:
You're right, didn't notice that. But it works anyway, weird.

But generally, I'm willing to go with your "original" approach because I can see how it is a better one. However, I don't know what to do with those items that should always appear.